### DISCUSSION TITLE
[Embed Dashboard] error on chart table

### DISCUSSION THREAD

**Discussion by Riskatrih** [apache/superset#19190] [Q&A / Help]
I'm using superset version 1.0.0 and python 3.8.1.
when embed a dashboard to the frontend app, I'm getting error on chart table like this
![image](https://user-images.githubusercontent.com/59104917/125889290-8fe143f2-7310-4b4f-b237-8a294b77b4ec.png)
but this error only applies to certain servers, on other servers there is no error or problem. 
What is this error usually caused by?
thankyou

**Comment by srinify** [comment]
Hey @Riskatri converting to a Discussion so hopefully someone in the community can help you debug this!
